onedrive_service/src/tests/test_service.py

```diff
@@ -0,0 +1,38 @@
+"""Test OneDriveService functionality."""
+# pylint: disable=protected-access
+
+import unittest
+
+import mock
+
+from onedrive_service.service import OneDriveService
+
+
+class TestOneDriveService(unittest.TestCase):
```

Comment:
Better to write tests using native `Py.Test` style tests.

Reply:
done